doc/release-notes-1.3.0.rst

```diff
@@ -0,0 +1,577 @@
+* New nRF9160 samples and libraries
+* Application update through external flash
+* General updates and bugfixes
+
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v1.3.0**.
+Check the ``west.yml`` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` for more information.
+
+
+Supported modem firmware
+************************
+
+This version of the |NCS| supports the following modem firmware for cellular IoT applications:
+
+* mfw_nrf9160_1.2.0
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See `Updating the nRF9160 DK cellular modem`_ for instructions.
+
+
+Change log
+**********
+
+The following sections provide detailed lists of changes by component.
+
+
+nRF9160
+=======
+
+* Added:
+
+  * :ref:`lib_date_time` library - maintains the current date-time UTC from multiple time sources (modem, NTP servers).
+  * :ref:`lib_ftp_client` library - can be used to download or upload FTP server files.
+  * :ref:`lib_nrf_cloud_agps` library - provides a way to request and process A-GPS data from nRF Cloud to be used with the nRF9160 SiP.
+    The :ref:`nrf_cloud_agps_sample` sample demonstrates how to use the library.
+  * :ref:`connectivity_bridge` application with dual UART and Bluetooth LE support - replaces the :ref:`usb_uart_bridge_sample` sample.
+
+* :ref:`gps_with_supl_support_sample` sample:
+
+  * Reworked the API to make it more flexible.
+  * Added an option to register an event handler.
+    All event types from the GPS will be received, as opposed to the previous single-trigger source.
+
+* :ref:`lib_nrf_cloud` library: (?)
+
+  * Added FOTA download progress monitoring.
+  * Added :ref:`lib_nrf_cloud_agps` support. (?)
+  * Added support for MQTT persistent sessions.
+  * Moved cloud connection polling out of the application into a separate nrf_cloud thread.
+  * Exposed MQTT topics to the nRF Cloud library.
+
+* :ref:`lib_aws_fota` library: (?)
+
+  * Switched from Zephyr's JSON parser to cJSON.
+  * Added support for MQTT persistent sessions.
+
+* :ref:`lib_fota_download` library:
+
+  * Added support for specifying a TCP port for connections.
+  * Added support for specifying the APN to be used.
+
+* :ref:`modem_key_mgmt` library:
+
+  * Added a :cpp:func:`modem_key_mgmt_cmp` function to the API, which allows to compare with a credential stored in the modem.
+  * Various minor fixes.
+
+* :ref:`modem_info_readme` library:
+
+  * Fixed an error in reading :cpp:enumerator:`MODEM_INFO_RSRP <modem_info::MODEM_INFO_RSRP>`.
+  * Added APN readout.
+
+* :ref:`lte_lc_readme` library:
+
+  * Added the concept of events.
+  * Allowed to modify PSM/eDRX parameters at runtime.
+  * Allowed to modify PDP context and PDN authentication at runtime.
+
+* :file:`lib/at_host` library: (?)
+
```

Comment:
remove (?)

lib/at_host does not have documentation it seems.

Reply:
Right, that's why we cannot link. But should the changes be removed? Or do the changes actually belong to a different library?

Reply:
ok